tools/cli/go-whisk/whisk/client.go

```diff
@@ -76,6 +77,7 @@ type Config struct {
     Verbose   	bool
     Debug       bool     // For detailed tracing
     Insecure    bool
+    CLIVersion  string
```

Comment:
The client should not know about the existence of a CLI as it is an abstraction.

Reply:
You can just have a UserAgent value here that can be modified by the CLI.